docs/en_US/Compression/CompressionReference.rst

```diff
@@ -1,16 +1,58 @@
-Python API Reference of Compression Utilities
-=============================================
+Compression Reference
+=====================
 
 .. contents::
 
-Sensitivity Utilities
+Compressors
+-----------
+
+Compressor
+^^^^^^^^^^
```

Comment:
it might be better to also add our supported pruners and quantizers

Reply:
thanks for the suggestion, added